charts/stardog/Chart.yaml

```diff
@@ -11,6 +11,6 @@ maintainers:
     url: https://www.stardog.com
 dependencies:
   - name: zookeeper
-    version: 5.5.1
+    version: 8.1.2
     repository: https://raw.githubusercontent.com/bitnami/charts/eb5f9a9513d987b519f0ecd732e7031241c50328/bitnami
```

Comment:
does this need to be updated?

Reply:
maybe I have my facts wrong but this is the only way we can update the chart, so i'd say yes.

Reply:
maybe i made my comment upon an incorrect statement. I meant that we do need to update the version, but after reading your comment again, I think you meant the repository.
yes, I think we can update the repository to the actual bitnami repo. the reason why we use this weird hash workaround, is because they dropped support for old versions on their main repo.
if we update to 8.1.2 then we might be good changing this repo back to `https://charts.bitnami.com/bitnami`
@ctffarley we should try that.

Reply:
Never mind, I forgot to run `helm dependency update` before testing. The chart version still isn't new enough to be in bitnami's repo.